tests/cpp17/generated_cpp17/optional_scalars_generated.h

```diff
@@ -928,6 +930,20 @@ inline const ::flatbuffers::TypeTable *ScalarStuffTypeTable() {
   return &tt;
 }
 
+template<>
+struct EnumTraits<OptionalByte> {
+    constexpr static auto type_table = OptionalByteTypeTable;
+    constexpr static auto name = EnumNameOptionalByte;
+};
+
+inline EnumTraits<OptionalByte> EnumValTraits(OptionalByte) {
```

Comment:
since this function only has one user currently, is it worth the clutter? Can we just call `EnumTraits<decltype e>::name` or similar below?

Doesn't even need an instance because its static?